public/locales/de/translations.json

```diff
@@ -0,0 +1,49 @@
+{
+	"root.description": "Eine Alternative zu dem Nix Ökosystem",
+
+	"header.community": "Gemeinschaft",
+
+	"goals.title": "Die Ziele",
```

Comment:
In a similar spirit to the 'Unsere Werte' suggestion below: 'Ziele' / 'Unsere Ziele'?

Reply:
+1 for "Unsere Ziele"

Reply:
Yes